src/graphs.jl

```diff
@@ -266,6 +266,72 @@ function edgifyrates(rxs, specs)
     es
 end
 
+# create edges from one reaction complex to another reaction complex
+function edgifycomplex(δ)
+    attr = Attributes()
+    return map(δ) do p
+        return Edge([p[1], p[2]] , attr)
+    end
+end
+
+# modify vector of string of complexes into graphviz compatible strings
+function modifystrcomp(strcomp::Vector{String})
+  for i in 1:length(strcomp)
+    #   cannot allow + as per graphviz
+      if occursin("+",strcomp[i])
+          strcomp[i] = replace(strcomp[i], "+" => "₊");
+      end
+    #   cannot allow (t) as per graphviz
+      if occursin("(t)",strcomp[i])
+          strcomp[i] = replace(strcomp[i], "(t)" => "")
+      end
+      #   prettify NUll with ∅
+      if occursin("0",strcomp[i])
+          strcomp[i] = replace(strcomp[i], "0" => "∅")
+      end
+  #   cannot allow spaces as per graphviz
+      strcomp[i] = filter(x -> !isspace(x),strcomp[i])
+  end
+#   numbers cannot be at the start of string as per graphviz,
+#  so lets prettify by prefixing complex as  "ℂ∷""
+  strcomp = "ℂ∷".*strcomp
```

Comment:
Is there no way around this? The docs mention HTML strings or putting quotes around stings. Do either of these get around the issue of not having spaces or a leading number?

Reply:
> Is there no way around this? The docs mention HTML strings or putting quotes around stings. Do either of these get around the issue of not having spaces or a leading number?

I am not sure. I was searching for a way around by using HTML-like strings in graphviz, `graphs.jl` mentions following
https://github.com/SciML/Catalyst.jl/blob/bfe0ce34f54759c1a6f76ae09f1c2e906784b0ff/src/graphs.jl#L21
 I will test more